core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -586,6 +586,20 @@ public String toString() {
           .setScope(Scope.SERVER)
           .setIsHidden(true)
           .build();
+  public static final PropertyKey EXIT_COLLECT_METRICS =
```

Comment:
can we think about merging the properties? I am concerned we have too many configuration properties

Reply:
Ok I will collapse some of properties but note this: I intend to leave all these switches turned ON, and the properties are just used to turn them off. I figure this dumping happens when the master/worker is not serving so there's low risk even if the dumping is slow or fails for any reason.

Given the output is to a separate file, on top of my head i can't think of a use case these dumping needs to be turned off. Maybe users want the dumping to be off when a failover happens. Maybe users want to have metrics but not jstacks. But these are just my imagination so I'm fine with not opening many properties as options, until users really ask for that.